RELEASING.md

```diff
@@ -1,4 +1,19 @@
-# Steps for releasing `matrix-sdk-crypto-wasm`
+# Releasing `matrix-sdk-crypto-wasm`
+
+## Before you release
+
+Assuming you are making a release to get the latest Rust code, you should bump
+the version of `matrix-rust-sdk` we are depending on in `Cargo.toml`.
```

Comment:
it's `Cargo.lock` that needs the update, not `Cargo.toml`.

Reply:
Updated.